README.md

```diff
@@ -30,11 +30,7 @@ which has been tested to use from this exporter:
 Build
```

Comment:
Think we'd better to have separate section: `Install`? 
Especially for go beginners, a way of start developing go project is little tricky I think :D